docs/sources/reference/commandline/cli.md

```diff
@@ -349,6 +350,31 @@ https://linuxcontainers.org/) via the `lxc` execution driver, however, this is
 not where the primary development of new functionality is taking place.
 Add `-e lxc` to the daemon flags to use the `lxc` execution driver.
 
+### Daemon user namespace support
```

Comment:
Love the table Phil!  I took a closer look at this content from the reader perspective. I came up with some more questions and a bit of an edit. If it is too early in the PR to do this, just ignore me. 

----

## Remap the container's root user

You can remap a container's root user to an unprivileged user on the Docker
host. To allow remapping, you enable Linux user namespace support on the Docker
daemon. The daemon's namespace configuration applies to all containers the
daemon runs; You cannot remap on a per-container basis.  

To enable user namespace support, provide a `username` or Linux `uid:gid` to the
`-root` flag. If you want the daemon to create and use a Docker default user
management, specify `default` instead. When you specify `default`, the daemon
creates a user and group named `dockroot` (if they don't already exist). Then,
the daemon uses `dockerroot` root as the root remapped ID inside all containers
for that daemon instance.


----


I removed this clause.

"Due to the requirement that the filesystem layers need to have user/group
ownership modification to make this remapping useful,"

A very complex cause -- I think you are trying to explain to the reader why you
are remapping at the daemon level not per container.  Do they need the
explanation to use the feature? I tend to think this is explanation is unnecessary.

Also, if a user supplies just a `username` what happens to the gid/group
ownership values. Is the user both the "user" and the "group" or is the group
owner still root?  Maybe this goes back to your clause but that is immediately
clear.

Finally, just as a nice to have, why would the user want to use the default user
management versus enabling remapping? Is it so obvious or could we do a little
throw away line that would clue newbs into the feature?